data.sql

```diff
@@ -42,3 +42,38 @@ UPDATE animals SET owner_id = (SELECT id FROM owners WHERE full_name='Bob' ) WHE
+INSERT INTO visits (animal_id, vet_id, date) VALUES ((SELECT id FROM animals WHERE name='Pikachu'),(SELECT id FROM vets WHERE name = 'Maisy Smith'), '2020-01-05');
+
+INSERT INTO visits (animal_id, vet_id, date) VALUES ((SELECT id FROM animals WHERE name='Pikachu'),(SELECT id FROM vets WHERE name = 'Maisy Smith'), '2020-03-08');
+
+INSERT INTO visits (animal_id, vet_id, date) VALUES ((SELECT id FROM animals WHERE name='Pikachu'),(SELECT id FROM vets WHERE name = 'Maisy Smith'), '2020-05-14');
+
+
+
+INSERT INTO visits (animal_id, vet_id, date) VALUES ((SELECT id FROM animals WHERE name='Devimon'),(SELECT id FROM vets WHERE name = 'Stephanie Mendez'), '2021-05-04');
+INSERT INTO visits (animal_id, vet_id, date) VALUES ((SELECT id FROM animals WHERE name='Charmander'),(SELECT id FROM vets WHERE name = 'Jack Harkness'), '2021-02-24');
+INSERT INTO visits (animal_id, vet_id, date) VALUES ((SELECT id FROM animals WHERE name='Plantmon'),(SELECT id FROM vets WHERE name = 'Maisy Smith'), '2019-12-21');
+INSERT INTO visits (animal_id, vet_id, date) VALUES ((SELECT id FROM animals WHERE name='Plantmon'),(SELECT id FROM vets WHERE name = 'William Tatcher'), '2020-08-10');
+INSERT INTO visits (animal_id, vet_id, date) VALUES ((SELECT id FROM animals WHERE name='Plantmon'),(SELECT id FROM vets WHERE name = 'Maisy Smith'), '2021-04-07');
+INSERT INTO visits (animal_id, vet_id, date) VALUES ((SELECT id FROM animals WHERE name='Squirtle'),(SELECT id FROM vets WHERE name = 'Stephanie Mendez'), '2019-09-29');
+INSERT INTO visits (animal_id, vet_id, date) VALUES ((SELECT id FROM animals WHERE name='Angemon'),(SELECT id FROM vets WHERE name = 'Jack Harkness'), '2020-10-03');
+INSERT INTO visits (animal_id, vet_id, date) VALUES ((SELECT id FROM animals WHERE name='Angemon'),(SELECT id FROM vets WHERE name = 'Jack Harkness'), '2020-11-04');
+INSERT INTO visits (animal_id, vet_id, date) VALUES ((SELECT id FROM animals WHERE name='Boarmon'),(SELECT id FROM vets WHERE name = 'Maisy Smith'), '2019-01-24');
+INSERT INTO visits (animal_id, vet_id, date) VALUES ((SELECT id FROM animals WHERE name='Boarmon'),(SELECT id FROM vets WHERE name = 'Maisy Smith'), '2019-05-15');
+INSERT INTO visits (animal_id, vet_id, date) VALUES ((SELECT id FROM animals WHERE name='Boarmon'),(SELECT id FROM vets WHERE name = 'Maisy Smith'), '2020-02-27');
+INSERT INTO visits (animal_id, vet_id, date) VALUES ((SELECT id FROM animals WHERE name='Boarmon'),(SELECT id FROM vets WHERE name = 'Maisy Smith'), '2020-08-03');
+INSERT INTO visits (animal_id, vet_id, date) VALUES ((SELECT id FROM animals WHERE name='Blossom'),(SELECT id FROM vets WHERE name = 'Stephanie Mendez'), '2020-05-24');
+vet_clinic=# INSERT INTO visits (animal_id, vet_id, date) VALUES ((SELECT id FROM animals WHERE name='Blossom'),(SELECT id FROM vets WHERE name = 'William Tatcher'), '2021-01-11');
+
```

Comment:
- [ ] Good job so far 👏 👏. It looks like **this last data was not recorded as It looks commented**. Please uncomment it and make sure that the data is recorded in the Postgresql database so that they will be taken into consideration while doing queries 😃 👍 

HINT 💡 
```sql
INSERT INTO visits (animal_id, vet_id, date) VALUES ((SELECT id FROM animals WHERE name='Blossom'),(SELECT id FROM vets WHERE name = 'William Tatcher'), '2021-01-11');
```